pennylane/devices/execution_config.py

```diff
@@ -54,6 +54,15 @@ class ExecutionConfig:
     True indicates a request to either use the device gradient or fail.
     """
 
+    use_device_jacobian_product: Optional[bool] = None
+    """Whether or not to use the device provided vjp or jvp to compute gradients.
+
+    ``None`` indicates to use the device if possible, but to fall back to the device Jacobian
+    or PennyLane behavior if it isn't.
```

Comment:
```suggestion
    or PennyLane behaviour if it isn't.
```
Sorry - very picky but this is to follow :canada: English as recommended in the style guide.